RL dashboard updates (#2903)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md

In the Admin Console, go to **Reports** > **Rate Limits**. The rate limits report opens in the **Overview** tab.

> ![Displays the rate limits dashboard to monitor and browse rate limit usage](/img/rl_dashboard6.png)

Reviewer: ```suggestion
![Displays the rate limits dashboard to monitor and browse rate limit usage.](/img/rl_dashboard6.png)
```

The Violations line graph plots the number of violations over the selected time period, which can be the last 24 hours, last 7 days, last month, or last 3 months. The Violations per API bar graph shows the top ten number of violations by API over the same time period.

> ![Displays the violations graph and and bar graph to show violations over a period of time](/img/rl_dashboard7.png)

Reviewer: ```suggestion
![Displays the violations over a period of time graph and the top 10 endpoint violations bar graph.](/img/rl_dashboard7.png)
```

You can hover over the line graph or bar graph to get more details. The details also include a link to the system log where you can view filtered events specific to that API and time period.

> ![Displays details on the violations graph and bar graph](/img/rl_dashboard8.png)

Reviewer: ```suggestion
![Displays details on the violations graph.](/img/rl_dashboard8.png)
```
I don't see the bar graph on this image?

#### APIs table

The APIs table shows each API and its corresponding rate limit for your org. The rate limits include all overrides, dynamic scale, or Workforce multipliers. The table also shows the number of violations and warnings for each API in the last hour, and includes a trendline of usage in the last hour.

> ![Displays the APIs table for each API](/img/rl_dashboard9.png)

Reviewer: ```suggestion
![Displays the APIs table with corresponding rate limits for each API.](/img/rl_dashboard9.png)
```
